packages/fxa-payments-server/package.json

```diff
@@ -15,7 +15,7 @@
     "eject": "react-scripts eject",
     "test": "npm-run-all test:*",
     "test:frontend": "react-scripts test --coverage --verbose",
-    "test:server": "jest --coverage --verbose --config server/jest.config.js",
+    "test:server": "FLOW_ID_KEY=testo jest --coverage --verbose --config server/jest.config.js",
```

Comment:
This looks like something that should get added to mocks in server tests. Shouldn't need to configure the environment for jest

Reply:
@lmorchard Thanks for looking at this PR.  It was created with the intention of sending the metrics directly to content server.  Now that's no longer the case; we'll build on the server code that's been added for Amplitude metrics.  And I think the request from the frontend will also be different and simpler.

tl;dr I'll add a [HOLD] to the PR title now.